mobx_codegen/lib/src/template/store.dart

```diff
@@ -56,6 +60,30 @@ abstract class StoreTemplate {
 
   $actions''';
 
+    if (generateToString) {
+      for(var i = 0; i < observables.templates.length; i++){
```

Comment:
Why not use a map instead of an imperative loop ?

```dart
observables.templates.map ((current){ /*... */ })
```

Reply:
done